frontend/src/views/NavigationDefault.vue

```diff
@@ -12,17 +12,17 @@
     v-else app
     fixed grow>
     <v-btn :to="{name: 'home'}">
-      <span>Home</span>
+      <span>{{ $tc('components.navigation.home') }}</span>
       <span>
         <v-icon>$vuetify.icons.ecamp</v-icon>️
       </span>
     </v-btn>
     <v-btn :to="{name: 'camps'}">
-      <span>Meine Camps</span>
+      <span>{{ $tc('components.navigation.myCamps') }}</span>
       <v-icon>mdi-format-list-bulleted-triangle</v-icon>
     </v-btn>
     <v-btn :to="{name: 'profile'}">
-      <span>Profile</span>
+      <span>{{ $tc('components.navigation.profile') }}</span>
```

Comment:
Key should be `views.navigationDefault.profile` (also remember to change it in the JSON)